www/grpc/proto/network.proto

```diff
@@ -14,7 +14,10 @@ service Network {
 }
 
 // Request message for retrieving overall network information.
-message GetNetworkInfoRequest {}
+message GetNetworkInfoRequest {
+  // Only returns the peers with online status
+  bool only_online = 1;
```

Comment:
`only_connected` makes more sense because other peers may be online but not connected to this peer.